### DISCUSSION TITLE
Stuttering/missing frames video RC4 vs RC5

### DISCUSSION THREAD

**Discussion by Minglarn** [blakeblackshear/frigate#1894] [Ask A Question]
Hi! 
Running 1049673 at the moment. Yesterday I updated the image file to rc5 just too keep up with all the changes...
Today when I was inspecting the recorded videos they where all stuttering and a lot of frames was missing. 

Are there any significant changes in how FFMPEG records clips in rc5?

**Comment by blakeblackshear** [comment]
What hardware are you using?
